### PR TITLE
TTLauncherItem userinfo

### PR DESCRIPTION
**Pull request by taknology**
Updated TTLauncherItem to include userInfo property for assignment of user specific information similar to userInfo property to table view cells in Three20.

### CODE REVIEW

**Comment by jwang** [comment]
thanks!

File: src/Three20UI/Sources/TTLauncherItem.m [hunk 1]
```diff
@@ -32,6 +32,7 @@
 @synthesize style       = _style;
 @synthesize badgeValue  = _badgeValue;
 @synthesize canDelete   = _canDelete;
+@synthesize userInfo	= _userInfo;
```
Comment (jwang) [inline src/Three20UI/Sources/TTLauncherItem.m]: spacing issue. use the lint too. I'll fix it this time around though.

File: src/Three20UI/Headers/TTLauncherItem.h [hunk 2]
```diff
@@ -30,8 +30,10 @@
   NSString*       _badgeValue;
 
   BOOL            _canDelete;
+    id			  _userInfo;
 }
 
+@property (nonatomic, assign) id userInfo;
```
Comment (jwang) [inline src/Three20UI/Headers/TTLauncherItem.h]: spacing again. needs linting. :)

**Comment by taknology** [comment]
Sorry John.  Want me to pull this back and fix it?

**Comment by jwang** [comment]
no worries. i took care of it. next time, just be sure to run with the Lint tool turned on. It's supposed to catch these things. If it's not catching them, then jeff and i need to fix it. :)

**Comment by taknology** [comment]
Thanks John.  I am working on sharing a bunch of other modifications I have made.  I read Jeff's email on the linter tool he sent a short time ago.  Do I need to do something to get it to work automatically?  I don't see that in the email and Option-D does not appear to be doing anything.  I'm on Xcode 4 if that makes a difference.

**Comment by taknology** [comment]
Scratch that.  It appears it did run.  Here is the output for the linter for the Three20UI project that includes my change.  It's not reporting any issues.

PhaseScriptExecution Lint ../../Build/Three20UI.build/Debug-iphonesimulator/Three20UI.build/Script-668E65E7131ADFCA00ACE4AB.sh
    cd /Users/tonylewis/Repositories/three20/src/Three20UI
    /bin/sh -c /Users/tonylewis/Repositories/three20/src/Three20UI/../../Build/Three20UI.build/Debug-iphonesimulator/Three20UI.build/Script-668E65E7131ADFCA00ACE4AB.sh

**Comment by jverkoey** [comment]
Oh no, looks like this got pulled into https://github.com/facebook/three20/pull/470 as well. To avoid this make sure to push each change as a separate branch in the future :) Not a big deal, just clutters up the code reviews a bit.

**Comment by jverkoey** [comment]
Looks good to me.

**Pull request closed without merging**